aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -560,10 +562,10 @@ void onChallengeResponse(final long correlationId, final long clusterSessionId,
 
                 if (session.id() == clusterSessionId && session.state() == CHALLENGED)
                 {
-                    final long now = clusterClock.time();
-                    final long nowMs = clusterTimeUnit.toMillis(now);
-                    session.lastActivityNs(clusterTimeUnit.toNanos(now), correlationId);
-                    authenticator.onChallengeResponse(clusterSessionId, encodedCredentials, nowMs);
+                    final long nowNs = clusterClock.timeNanos();
+                    final long nowMillis = NANOSECONDS.toMillis(nowNs);
```

Comment:
Should be `nowMs`.

Reply:
I had initially named it `nowMs` but thought it looked too much like `nowNs`.  Will change it back to nowMs if that is preferred.